nix/default.nix

```diff
@@ -113,6 +117,11 @@ in
         }'
       '';
 
+      postInstall = lib.optional wrapRuntimeDeps ''
```

Comment:
I think this needs a rewrite now that I've merged https://github.com/hyprwm/Hyprland/commit/78826c6d18ee587a28cf39479f9a015888d2e096.

Reply:
Yep looks like it